siteorigin-panels.php

```diff
@@ -129,6 +129,19 @@ public static function is_panel( $can_edit = false ) {
 		$is_panel =  ( siteorigin_panels_is_home() || ( is_singular() && get_post_meta(get_the_ID(), 'panels_data', false) ) );
 		return $is_panel && ( ! $can_edit || ( ( is_singular() && current_user_can( 'edit_post', get_the_ID() ) ) || ( siteorigin_panels_is_home() && current_user_can('edit_theme_options') ) ) );
 	}
+	
+	/**
+	 * Check if the user is currently using the live editor.
+	 *
+	 * @return bool
+	 */
+	public static function  is_live_editor(){
```

Comment:
This could be done a lot more efficiently with just

`return ! empty( $_GET['siteorigin_panels_live_editor'] )`